content/blog/2023/06 /02

```diff
@@ -0,0 +1,17 @@
+title: "The journey of Letica and development assessments - Research Call with Jonathan Reams"
+created: 2023-06-02
+categories: 
+  - "research-call"
+authors: 
+  - Rufuspollock
+
+>This call forms part of our monthly Research Call events. To get involved with this and our community calls please join the Life Itself [Whatsapp Chat](https://chat.whatsapp.com/JNJCTZugNQn)
+
+**When:** 15:00CET, Friday June 9th, 2023
+**Who:** Jonathan Reams is an Associate Professor at the Norwegian University of Science and Technology and has an insatiable curiosity about the essence of human nature and how to cultivate this essence in the service of leadership. He is also one of the leading experts on Letica in Europe. Jonathan will discuss the journey so far for Lectica over the past 20 years, along with his experience in applying Lectica, and the future vision of the application of developmental assessments, before opening out the call into a wider discussion.
+**What:** In this call Jonathan will present on his experience with Letica, the learning and development assessment model. 
```

Comment:
I think he will share more broadly but this is fine. Also i'm not sure many people would know what Lectica is so maybe need to link to it.